tests/end-to-end/train.sh

```diff
@@ -28,7 +28,7 @@ check_status $? "Training run not succesful!"
 echo ""
 echo "Test classification mode of training"
 echo ""
-python $root_dir/main.py train \
+atacworks train \
```

Comment:
do any of the READMEs or ipython notebooks need to be updated with this as well?

Reply:
updated the notebooks. they were merged this morning :)